w3-hw1-ThuyDung/cypress/pages/bookApplication/bookPage.js

```diff
@@ -0,0 +1,36 @@
+export const bookPage= {
+    TXT_SEARCH : "#searchBox",
+    BTN_BOOK :".action-buttons",
+    HEADER: ".main-header",
+    LBL_BOOK: "#title-wrapper > div.col-md-9.col-sm-12 label",
+    BTN_ADDBOOK : "div.text-right.fullButton",
+
+    isNavigateToBookStore(){
+        cy.get(this.HEADER).should('have.text', "Book Store");
+        return this;
+    },
+    findBookByTyping(name){
+        cy.get(this.TXT_SEARCH).type(name);
+        cy.get(this.BTN_BOOK).contains(name).eq(0).click();
```

Comment:
What is the purpose of `eq(0)` ?

Reply:
When the user enters the name of the book they are searching for, there may be multiple books with the same name but different versions, so I added 'eq(0)' to ensure they select the first result